NotificationServiceExtension/NotificationService.Config.xcconfig

```diff
@@ -12,16 +12,17 @@
 // DEV | STAGING | PROD
 SERVICE_ENV = STAGING
 
-// RINKEBY | MAINNET
-NETWORK = RINKEBY
-
 // Bundle Identifier
-_APP_BUNDLE_ID_PROD_MAINNET    = io.gnosis.multisig.prod.mainnet.NotificationServiceExtension
```

Comment:
OMG. I forgot that we have this config 🤦🏻

Reply:
yes, I missed it